Twig/PagerfantaExtension.php

```diff
@@ -103,6 +103,7 @@ private function createRouteGenerator($options = array())
                 'routeName'     => null,
                 'routeParams'   => array(),
                 'pageParameter' => '[page]',
+                'omitFirstPage' => true
```

Comment:
Did not do that, as I was not sure if the revert was desired. However, I think it is great to do so, and it is already merged see :)